opm/simulators/linalg/ExtraSmoothers.hpp

```diff
@@ -0,0 +1,42 @@
+#ifndef OPM_EXTRASMOOTHERS_HPP
+#define OPM_EXTRASMOOTHERS_HPP
+
+#include "DILU.hpp"
+
+namespace Dune
+{
+  template <class M, class X, class Y>
```

Comment:
Indent same as next line. I see also that the next namespace is placed on the same level, which is (I think) part of our clang-format style, so it will be a little weird anyway, but less weird is better...

Reply:
For some reason VSCode used 2 spaces for this file only... Fixed